aws_marketplace/curating_aws_marketplace_listing_and_sample_notebook/Algorithm/Sample_Notebook_Template/title_of_your_product-Algorithm.ipynb

```diff
@@ -0,0 +1,867 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 2. Prepare dataset"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<font color='red'> For Seller to update: Add all necessary imports in following cell. \n",
+    "If you need specific packages to be installed, try to provide them in this section, in a separate cell. </font>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import base64\n",
+    "import json \n",
+    "import uuid\n",
+    "from sagemaker import ModelPackage\n",
+    "import sagemaker as sage\n",
+    "from sagemaker import get_execution_role\n",
+    "from sagemaker import ModelPackage\n",
+    "from urllib.parse import urlparse\n",
+    "import boto3\n",
+    "from IPython.display import Image\n",
+    "from PIL import Image as ImageEdit\n",
+    "import urllib.request\n",
+    "import numpy as np"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### A. Dataset format expected by the algorithm"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<font color='red'>For Seller to update: In following cell, please provide a description of the dataset accepted by the training job. This will help customer understand what the dataset fed to the algorithm needs to look like. <font>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": []
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You can also find more information about dataset format in **Usage Information** section of <font color='red'> For Seller to update:[Title_of_your_product](Provide link to your marketplace listing of your product).</font>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### B. Configure and visualize train and test dataset"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<font color='red'>For Seller to update: upload the sample training dataset into data/train directory and update the training_dataset parameter value in following cell. You are strongly recommended to either upload the dataset into data/train directory or download it from a reliable source at runtime. **If you intend to download it at run-time, please add relevant code in following cell.** Do not hardcode your bucketname. </font>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "training_dataset='data/train/<FileName.ext>'"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<font color='red'>For Seller to read: You are recommended to support test channel and accept a test dataset to calculate your algorithm metrics on. Emit both - training as well as test metrics.</font>\n",
```

Comment:
```suggestion
    "<font color='red'>For Seller to read: We recommend that you support a test channel and accept a test dataset to calculate your algorithm metrics on. Emit both - training as well as test metrics.</font>\n",
```
This could be rephrased for clarity.